mlflow/store/tracking/dbmodels/models.py

```diff
@@ -1981,6 +1981,11 @@ class SqlJob(Base):
     Job retry count: `Integer`
     """
 
+    last_update_time = Column(BigInteger(), default=get_current_time_millis)
```

Comment:
is this field nullable?

Reply:
It always has a value, when created, it is filled with creation time.

Reply:
I see, so it is fine to accidentally set it to none?

Reply:
> it is fine to accidentally set it to none?

it's fine.